notify/opsgenie/opsgenie.go

```diff
@@ -297,3 +272,50 @@ func (n *Notifier) createRequests(ctx context.Context, as ...*types.Alert) ([]*h
 
 	return requests, true, nil
 }
+
+func (n *Notifier) updateAlert(ctx context.Context, tmpl func(string) string, key notify.Key, alias string, requests *[]*http.Request) (bool, error) {
```

Comment:
(suggestion) rather than passing a pointer to a slice, it would be more idiomatic to return a slice and let the caller append it to the existing slice (e.g. `requests = append(requests, additionalRequests...)`)
```suggestion
func (n *Notifier) updateAlert(ctx context.Context, tmpl func(string) string, key notify.Key, alias string) ([]requests, bool, error) {
```

Reply:
good idea!